src/test/groovy/org/prebid/server/functional/model/db/StoredRequest.groovy

```diff
@@ -34,8 +34,10 @@ class StoredRequest {
         new StoredRequest(reqid: ampRequest.tagId, accountId: ampRequest.account, requestData: bidRequest)
     }
 
-    static StoredRequest getDbStoredRequest(BidRequest bidRequest, BidRequest storedRequest) {
-        new StoredRequest(reqid: bidRequest.ext.prebid.storedRequest.id, accountId: bidRequest.site.publisher.id,
+    static StoredRequest getDbStoredRequest(BidRequest bidRequest,
+                                            String accountId = bidRequest.site.publisher.id,
```

Comment:
1) Arguments with default values should go last unless really need to lessen possible confusion.
2) Please use null-safe reference.